src/initializeCacheClient.js

```diff
@@ -0,0 +1,35 @@
+/* Memcached cache client. */
+import { get } from 'lodash';
+import { bind, getContainer, getMetadata } from '@globality/nodule-config';
+
+import Cache from './memcached-promisify';
+
+function createCacheClient() {
+    const { config } = getContainer();
+    const memcachedConfig = get(config, 'cache.memcached');
+    const cacheHost = get(config, 'cache.hosts', '').split(',');
+    const client = new Cache(
+        { cacheHost },
+        memcachedConfig,
+    );
+
+    return client;
+}
+
+/* Configure caching for a nodule-graphql framework.
+ */
+
+export default function initializeCacheClient() {
+    const { config, terminal } = getContainer();
+    // fix-up caching configuration
+    if (!get(config, 'cache.enabled')) {
+        if (!getMetadata().testing) {
+            terminal.disabled('caching');
+        }
+    } else {
+        if (!getMetadata().testing) {
+            terminal.enabled('caching');
+        }
+        bind('cache', () => createCacheClient());
```

Comment:
Should we make the bind string ("cache") configurable?

Reply:
I don't see why

Reply:
if we're going to use another `cache` service

Reply:
we'd put it in the same place. why would we use multiple providers for a single gateway? I don't think there's a reason for solving a problem which doesn't exist...